UI/GameSettingsScreen.cpp

```diff
@@ -585,6 +585,16 @@ void GameSettingsScreen::CreateViews() {
 	networkingSettings->Add(new CheckBox(&g_Config.bEnableAdhocServer, n->T("Enable built-in PRO Adhoc Server", "Enable built-in PRO Adhoc Server")));
 	networkingSettings->Add(new ChoiceWithValueDisplay(&g_Config.sMACAddress, n->T("Change Mac Address"), nullptr))->OnClick.Handle(this, &GameSettingsScreen::OnChangeMacAddress);
 	networkingSettings->Add(new PopupSliderChoice(&g_Config.iPortOffset, 0, 60000, n->T("Port offset", "Port offset(0 = PSP compatibility)"), 100, screenManager()));
+	networkingSettings->Add(new ItemHeader(ms->T("Remote Disc Streaming")));
+	networkingSettings->Add(new CheckBox(&g_Config.bRemoteISOManual , n->T("Manual Mode Client", "Manual Mode Client")));
```

Comment:
I agree it should be on the UI for connecting.  I didn't know how to code that with the screen manager, so I put it here.  The remoteisoscreen has 2 columns and I didn't know how to put a linear layout below that spans the entire screen.